plugins/woocommerce/tests/performance/requests/api/orders.js

```diff
@@ -84,7 +123,7 @@ export function ordersAPI() {
 		);
 		check( response, {
 			'status is 201': ( r ) => r.status === 201,
-			"body contains: 'Pending' Status": ( response ) =>
```

Comment:
That is [valid syntax](https://grafana.com/docs/k6/latest/javascript-api/k6/check/), I don't see why you need to remove it.

Reply:
It works in both cases when tested locally (probably will pass also on CI), but I'm getting linting issues in CI that the response is already declared in file, as a `let response;`. Please check. @adimoldovan

Reply:
Yes, that's because of the global `response`. But you don't have to delete it, only rename it. You can use `r`, just like in the above `'status is 201'` check.

Reply:
Ah I see, done in a new commit.